main.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 var (
-	VERSION = "0.0.1"
+	VERSION = "0.0.17"
```

Comment:
Don't need to change the VERSION variable manually since it should be changed while compiling the binary. You can run `./bin/local-path-provisioner --version` to verify.